package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@artsy/library-example",
+  "private": true,
```

Comment:
We should maybe make sure this actually works?

Reply:
Need to remove this